src/lumen_anndata/source.py

```diff
@@ -44,12 +49,31 @@ class AnnDataSource(DuckDBSource):
         doc="An AnnData instance or path to a .h5ad file to load.",
     )
 
-    ephemeral = param.Boolean(default=True, doc="Always ephemeral (in-memory) by virtue of AnnDataSource.")
-
     filter_in_sql = param.Boolean(default=True, doc="Whether to apply filters in SQL or in-memory.")
 
     source_type = "anndata"
 
+    _opened = {}  # Track files: {filename: (adata_object, is_temporary)}
+
```

Comment:
```suggestion
operations = param.List(default=[], doc="List of AnnDataOperation instances to apply to the data.")
```

This appears to be missing and fixes the error in [my comment ](https://github.com/holoviz-topics/lumen-anndata/pull/11#issuecomment-2963086515)